### PR TITLE
docs(HMR): Fix code example

### PR DESCRIPTION
**Pull request by RobbertWolfs**
Updating code example in guides.
Guides told to remove the `print.js` but it deleted two lines and putted back one line (same one as one of the deleted). So I just deleted the one line that was necessary

### CODE REVIEW

File: src/content/guides/hot-module-replacement.md [hunk 1]
```diff
@@ -45,9 +46,8 @@ __webpack.config.js__
 
   module.exports = {
     entry: {
--      app: './src/index.js',
+       app: './src/index.js',
```
Comment (asapach) [inline src/content/guides/hot-module-replacement.md]: you have a trailing comma now

File: src/content/guides/hot-module-replacement.md [hunk 2]
```diff
@@ -45,9 +46,8 @@ __webpack.config.js__
 
   module.exports = {
     entry: {
--      app: './src/index.js',
+       app: './src/index.js'
```
Comment (asapach) [inline src/content/guides/hot-module-replacement.md]: now compare this line: you've changed it, but it's not reflected in the diff
does this PR make sense?

Comment (RobbertWolfs) [reply src/content/guides/hot-module-replacement.md]: The following sentence is written in the markdown file : "We'll also remove the entry point for print.js..."

So I guess this should be the correct example 
screen shot 2017-12-15 at 22 36 48


And this is the current example
screen shot 2017-12-15 at 22 38 29

Which removes the `app.js` line and then re-adds it.. Which is a step to much I guess

File: src/content/guides/hot-module-replacement.md (hunk 1 above)
Comment (RobbertWolfs) [reply src/content/guides/hot-module-replacement.md]: fixed

**Comment by asapach** [comment]
That's how diff works: when you change a line it shows the old text as "removal" and the new text as an "addition". By removing the trailing comma in the first line you've change it, which needs to be reflected in the diff.

**Comment by RobbertWolfs** [comment]
Ok, thought it was weird but.. ignore this PR then

**Pull request closed without merging**